README.md

```diff
@@ -4,92 +4,108 @@
 ## Build Status
 [![Build Status](https://travis-ci.org/WorldBank-Transport/DRIVER-Android.svg?branch=develop)](https://travis-ci.org/WorldBank-Transport/DRIVER-Android)
 
-## Developing
-See the steps under [Setup](#setup), below. If you need to deploy to an existing stack or set up a new stack
-then you should read this whole document thoroughly.
+## Getting Started
+To set up Android Studio to build and run this project, there are a few steps you need to perform first.
 
-## Generating a Keystore
-A valid keystore is required for most of the remaining steps. If you are developing an app that is
-targeted at an existing installation of DRIVER, then that installation should have a pre-existing
-keystore that you can use.
+- Generate a keystore
```

Comment:
This is not strictly necessary, but you may want to have these items link to the corresponding section within the document, like a table of contents.

For reference: https://gist.github.com/asabaylus/3071099